test/test_aes.py

```diff
@@ -94,6 +97,19 @@ def test_decrypt_text(self):
         decrypted = (aes_decrypt_text(encrypted, password, 32))
         self.assertEqual(decrypted, self.secret_msg)
 
+    def test_ecb_encrypt(self):
+        data = bytes_to_intlist(self.secret_msg)
+        data += [0x08] * (BLOCK_SIZE_BYTES - len(data) % BLOCK_SIZE_BYTES)
```

Comment:
shouldn't this be done by the encryptor?

Reply:
oh
it should

also this doen't break AbemaTVIE, so I'll remove it

Reply:
this is because length of `self.secret_msg` isn't multiple of 16, but what should I pad in encryption menthod?

Reply:
idk. Is there no padding specs for it? If not, let it be

Reply:
usually PKCS#7 or `#5`, but nothing except CBC does padding in aes.py